docs/tutorials/tutorial_params_options_simulate.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>No need for a function and please separate the data generation from the plotting.</p><p> </p><p>Instead of looping over axis, you can do <code>axs = axs.flatten()</code> , then include <code>axs </code> in your <code>zip()</code> , remove <code>plt.subplot(a)</code> and use <code>ax.bar(...)</code>  instead of <code>plt.bar(...)</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>